USAGE.md

```diff
@@ -43,7 +43,7 @@ Each layer created is independent of other layers. However, layers can be
 <a href="#saving-and-loading-layers">saved locally</a>. Layer files are saved in easy to parse and easy to generate JSON
 file so that ATT&CK data can be used in other applications, analyzed beyond the capability of the ATT&CK Navigator, and
 generated by tools for import into the Navigator. The Layer file format is
-described <a href="layer-information-dialog">here</a>.
+described <a href="layers/LAYERFORMATv4_2.md">here</a>.
```

Comment:
Maybe this should just be the `/layers` folder instead of the specific file? Otherwise we will have to update the hyperlink when we update the layer format. It seems like that should work with your link replacement since it only cares about the `layers/` part.